src/moves_snaq.jl

```diff
@@ -38,6 +38,40 @@ function makeEdgeHybrid!(edge::Edge,node::Node,gamma::Float64; switchHyb=false::
     edge.containroot = false
 end
 
+# aux function to exchange who is the hybrid node
```

Comment:
Similarly here. This was removed when going to 1.0.0 and should stay removed. Some discussion around these functions [here ](https://github.com/JuliaPhylo/SNaQ.jl/pull/3#issuecomment-2744351441)